src/app/pages/signing/signing.component.html

```diff
@@ -169,3 +178,99 @@ <h5>Emojis</h5>
     </div>
   </div>
 </div>
+
+<!-- Picture not showing up? Modal -->
+<div class="modal fade" id="embed-modal" tabindex="-1" role="dialog" aria-labelledby="embed-modal-title"
```

Comment:
- `embed-modal` does not look good on mobile
- `embed-modal` should not be in dark theme